Sources/Logging/Strings/StoreKitStrings.swift

```diff
@@ -97,6 +99,10 @@ extension StoreKitStrings: CustomStringConvertible {
         case .no_cached_products_starting_store_products_request(let identifiers):
             return "No existing products cached, starting store products request for: \(identifiers)"
 
+        case let .sk1_payment_queue_too_many_transactions(count):
+            return "SKPaymentQueue sent \(count) updated transactions. " +
+            "This high number is unexpected and is likely due to bad state on your device."
```

Comment:
🤔 
I mean... this would probably be the sandbox account, right? like, if you use it a lot for tests, it's gonna just grow and grow. 
Apple updated so they only keep transactions for 90 days now, but if you open up an old sandbox account on a new device, I wouldn't be surprised if you got a ton of transactions. 

I would mostly recommend using a different sandbox account here

Reply:
for example if I test with one account a ton, then go to a new device and log in with that account, the new device will get all of those transactions coming in at once

Reply:
> I would mostly recommend using a different sandbox account here

That makes sense 👍🏻 I'll update the message to point to that.

Reply:
that looks great! One final nitpick: can we ensure that _that_ message is used only in sandbox, and have a different one for production? 
The production one could say something about there just being a very high number of transactions. 
I wouldn't necessarily call it _unexpected_, I can imagine that for a few apps they might actually have people with a ton of transactions (2 years of a weekly subscription is already over 100 renewals). I'd maybe just say something about there being a lot of transactions, so performance might be impacted

Reply:
Yup, good idea. I thought about it but I thought YAGNI because I had to inject `SystemInfo` to `StoreKit1Wrapper`. But I agree it's useful.

Reply:
@aboedo updated.